Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -173,8 +173,8 @@ public enum TypeDescription: Codable, Equatable, CustomStringConvertible {
   /// - Parameters:
   ///   - name: The simple name of the returned type.
   ///   - parent: The parent type for the returned type.
-  /// - Returns: Returns a type description of case `.member` with the given name as the name and the receiver as the base type.
-  /// - Note: This method only makes sense when the `parent` is of case  `simple`, `member`, `optional`, and `implicitlyUnwrappedOptional`.
+  /// - Returns: Returns a type description of case `.nested` with the given name as the name and the receiver as the base type.
```

Comment:
We return `.simple` in some cases. Should we update this comment?

Reply:
15e101be77005b904a638417a1c91d26638d877b